src/CoreRequest_Contended_Latency.cc

```diff
@@ -153,7 +154,7 @@ main(int argc, const char** argv) {
         CoreArbiterClient* client = CoreArbiterClient::getInstance();
         lowPriorityExec(client, lowPriorityRunning);
 
-        wait();
+        wait(NULL);
```

Comment:
This is fine, but I'm curious; what version of what compiler are you using?
In all the compilers, I've tried, using `wait()` with no arguments from `stdlib.h` would not cause any warnings or errors.